### PR TITLE
hardsuits supress hair fully again

### PR DESCRIPTION
**Pull request by optimumtact**

### CODE REVIEW

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -12,6 +12,7 @@
 	var/obj/item/clothing/suit/space/hardsuit/suit
 	item_color = "engineering" //Determines used sprites: hardsuit[on]-[color] and hardsuit[on]-[color]2 (lying down sprite)
 	actions_types = list(/datum/action/item_action/toggle_helmet_light)
+	dynamic_hair_suffix = ""
```
Comment (ShizCalev) [inline code/modules/clothing/spacesuits/hardsuit.dm]: Shouldn't be needed, it's already inheriting this from /obj/item/clothing/head/helmet/space

Comment (optimumtact) [inline code/modules/clothing/spacesuits/hardsuit.dm]: well I dunno why hardsuits are letting hair come through again then

Comment (ShizCalev) [inline code/modules/clothing/spacesuits/hardsuit.dm]: Are you sure the branch you're using is up to date?

Which hardsuits is it happening with?

Comment (optimumtact) [inline code/modules/clothing/spacesuits/hardsuit.dm]: it was happening on master with the CE hardsuit

Comment (ShizCalev) [inline code/modules/clothing/spacesuits/hardsuit.dm]: 3nb0tl3
3nb0u3k

Looks fine to me?

Comment (optimumtact) [inline code/modules/clothing/spacesuits/hardsuit.dm]: try with a ponytail, maybe the person who reported it bamboozled me

Comment (ShizCalev) [inline code/modules/clothing/spacesuits/hardsuit.dm]: Ponytails looking fine too

**Pull request closed without merging**